Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -153,6 +147,20 @@ class ProjectFileElements {
                              isReference: $0.isReference)
         })
 
+        // Local Packages
```

Comment:
I need to think about this further ... 

The inconsistency here is that all other dependencies are resolved via `allDependencyReferences` whereas this one isn't included there yet.